sdk/javascript/src/nem/models.js

```diff
@@ -2734,6 +2734,15 @@ class MultisigAccountModification {
 		this._cosignatoryPublicKeySize = 32; // reserved field
 	}
 
+	comparer() {
+		const { ripemdKeccak256 } = require('../utils/transforms'); // eslint-disable-line global-require
```

Comment:
hmm, doesn't eslint require two spaces prior to `//` ?

Reply:
no, only pylint requires that